Automattic/jetpack
# Pre-Commit: Run PHP lint before commit

I committed a PHP syntax error. While the unit testing caught it, we shouldn't rely on that. This would add a precommit action to lint if there were changed PHP files.

#### Changes proposed in this Pull Request:

* Add PHP lint on precommit.

#### Jetpack product discussion


n/a

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* On this branch, edit a PHP file with some type of bad syntax.
* Try to commit. Ensure it fails to commit.
